project/Dependencies.scala

```diff
@@ -511,7 +512,7 @@ object Dependencies {
 
   val servicesDependencies = List(
     "com.google.api" % "gax-grpc" % googleGaxGrpcV
-  )
+  ) ++ awsCloudDependencies
```

Comment:
Is it possible to make this a new project with its own set of dependencies rather than making the core `services` project rely on AWS?